### ISSUE TITLE
Функция тренировки модели train_model

### ISSUE THREAD

**Issue by GeekNekoS** [GeekNekoS/Chara#9]
Написать определение функции def train_model..
Аргументы:
directory: str - директория, где хранятся все папки с категориями фото
batch_size: int - размер батча
image_size: tuple[int, int] - размер фото, например, (224, 224)
Вывод:
None
1. Функция использует функцию load_train_test_val для загрузки датасетов. (уже написана)
2. Потом происходит создание модели с помощью функции create_model из таски №7 https://github.com/users/GeekNekoS/projects/7/views/1?pane=issue&itemId=85500455&issue=GeekNekoS%7CChara%7C4
3. Компиляция модели с помощью model.compile (keras)
4. Обучение происходит на train, валидация во время обучения на val. При обучении нужно использовать callback early_stopping.
5. Оценка результатов на датасете test. Будет функция для оценки обученности модели evaluate_model (ее кто-то другой напишет в другой таске, ее здесь писать не надо ). Собственно на выходе будет изображение графиков всяких и параметров accuracy, precision, recall, f1 и др
